src/NetworkCloud/docs/New-AzNetworkCloudCluster.md

```diff
@@ -0,0 +1,641 @@
+---
+external help file:
+Module Name: Az.NetworkCloud
+online version: https://learn.microsoft.com/powershell/module/az.networkcloud/new-aznetworkcloudcluster
+schema: 2.0.0
+---
+
+# New-AzNetworkCloudCluster
+
+## SYNOPSIS
+Create a new cluster or update the properties of the cluster if it exists.
+
+## SYNTAX
+
+```
+New-AzNetworkCloudCluster -Name <String> -ResourceGroupName <String>
+ -AggregatorOrSingleRackDefinitionNetworkRackId <String>
+ -AggregatorOrSingleRackDefinitionRackSerialNumber <String>
+ -AggregatorOrSingleRackDefinitionRackSkuId <String> -ClusterType <ClusterType> -ClusterVersion <String>
+ -ExtendedLocationName <String> -ExtendedLocationType <String> -Location <String> -NetworkFabricId <String>
+ [-SubscriptionId <String>] [-AggregatorOrSingleRackDefinitionAvailabilityZone <String>]
+ [-AggregatorOrSingleRackDefinitionBareMetalMachineConfiguration <IBareMetalMachineConfigurationData[]>]
+ [-AggregatorOrSingleRackDefinitionRackLocation <String>]
+ [-AggregatorOrSingleRackDefinitionStorageApplianceConfiguration <IStorageApplianceConfigurationData[]>]
+ [-AnalyticsWorkspaceId <String>] [-ClusterLocation <String>] [-ClusterServicePrincipalApplicationId <String>]
+ [-ClusterServicePrincipalId <String>] [-ClusterServicePrincipalPassword <String>]
```

Comment:
double-check all "password". Password should be "SecureString".

Reply:
Password fields are updated to use SecureSting.